packages/office-ui-fabric-react/src/components/Facepile/Facepile.scss

```diff
@@ -1,22 +1,32 @@
 @import '../../common/common';
 
+$spacingAroundItemButton: 4px;
+
 .root {
-  position: relative;
   width: auto;
 }
 
 .clear {
   clear: both;
 }
 
+.members {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  overflow: hidden;
+  margin: -1 * $spacingAroundItemButton;
+
+  > * {
```

Comment:
I did it this way because flex parent/flex child is a very coordinated effort between the two elements. The goal was to have this be more a "parent" driven concept, instead of child driven.

But I also know you don't use child selectors in your code base so...